packages/models/src/lib/category-config.ts

```diff
@@ -23,21 +23,22 @@ export const categoryRefSchema = weightedRefSchema(
 );
 export type CategoryRef = z.infer<typeof categoryRefSchema>;
 
-export const categoryConfigSchema = scorableSchema(
-  'Category with a score calculated from audits and groups from various plugins',
-  categoryRefSchema,
-  getDuplicateRefsInCategoryMetrics,
-  duplicateRefsInCategoryMetricsErrorMsg,
-)
-  .merge(
+export const categoryConfigSchema = z
+  .intersection(
+    scorableSchema(
+      'Category with a score calculated from audits and groups from various plugins',
+      categoryRefSchema,
+      getDuplicateRefsInCategoryMetrics,
+      duplicateRefsInCategoryMetricsErrorMsg,
+    ),
     metaSchema({
       titleDescription: 'Category Title',
       docsUrlDescription: 'Category docs URL',
       descriptionDescription: 'Category description',
       description: 'Meta info for category',
     }),
   )
-  .merge(
+  .and(
```

Comment:
Why change this intersections? Zod recommends merging objects in their [docs](https://zod.dev/?id=intersections):

> Though in many cases, it is recommended to use A.merge(B) to merge two objects. The .merge method returns a new ZodObject instance, whereas A.and(B) returns a less useful ZodIntersection instance that lacks common object methods like pick and omit.